MastodonSDK/Sources/MastodonLocalization/Generated/Strings.swift

```diff
@@ -1950,6 +1986,12 @@ public enum L10n {
         }
       }
     }
+    public enum FilteredNotificationBanner {
+      /// Plural format key: "%#@number_of_requests@"
+      public static func subtitle(_ p1: Int) -> String {
+        return L10n.tr("Localizable", "plural.filtered_notification_banner.subtitle", p1, fallback: "Plural format key: \"%#@number_of_requests@\"")
```

Comment:
When using `de_DE` I only see the fallback key here, this looks broken to me...

![Bildschirmfoto 2024-07-26 um 10 28 03](https://github.com/user-attachments/assets/bbc67e3e-041a-464e-ac41-43ecd4a35b8b)

Reply:
Yeah, I noticed that as well, as you can read [here](https://github.com/mastodon/mastodon-ios/pull/1319/commits/1bf763a94cfdde80f32ea4edb613a78efaeef5a8):

> For whatever reason the plural-translation doesn't have a fallback to Base?!

Once there's a (`de_DE`)-translation, it should be fixed  👍